package/ios/SurfaceUtils.mm

```diff
@@ -10,7 +10,6 @@ + (void)configureSurface:(CALayer *)layer
                     size:(CGSize)size
                contextId:(int)contextId {
   std::shared_ptr<rnwgpu::RNWebGPUManager> manager = [WebGPUModule getManager];
-  CGFloat scaleFactor = [UIScreen mainScreen].scale;
```

Comment:
This was not used, and this API is not supported on visionOS so I just removed it. 

If there is a need for scaleFactor its preferable to retrieve it using [UITraitCollection](https://developer.apple.com/documentation/uikit/uitraitcollection?language=objc)

Reply:
yes makes sense we don't do anything with this.